api/v1/server.gotmpl

```diff
@@ -123,25 +123,52 @@ var (
   tlsCACertificate  string
 )
 
+type SpecConfig struct {
```

Comment:
Could make this `ServerConfig` instead and move server-related options into this at some point?